x/stakeibc/keeper/hooks.go

```diff
@@ -270,9 +271,15 @@ func (k Keeper) TransferExistingDepositsToHostZones(ctx sdk.Context, epochNumber
 				continue
 			}
 			delegateAddress := delegateAccount.GetAddress()
-			// TODO(TEST-89): Set NewHeight relative to the most recent known gaia height (based on the LC)
 			// TODO(TEST-90): why do we have two gaia LCs?
-			timeoutHeight := clienttypes.NewHeight(0, 1000000)
+			blockHeight, found := k.GetLightClientHeightSafely(ctx, hostZone.ConnectionId)
+			if !found {
+				k.Logger(ctx).Error(fmt.Sprintf("Could not find blockHeight for host zone %s", hostZone.ConnectionId))
```

Comment:
Can we be a little more verbose here, something like 
```suggestion
				k.Logger(ctx).Error(fmt.Sprintf("Could not find blockHeight for host zone %s, aborting transfers to host zone this epoch", hostZone.ConnectionId))
```